src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,76 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String COMMA_SPLITTER = ",";
+    private static final String SPACE = " ";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String WRITE_EXCEPTION = "Can't write to file %s";
+    private static final String READ_EXCEPTION = "Can't read file with name %s";
+    private final int[] types = new int[2];
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String table = readFile(fromFileName);
+        String text = writeToString(processData(table));
+        writeToFile(text, toFileName);
+    }
+
+    private String readFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                stringBuilder.append(line).append(SPACE);
```

Comment:
Is `SPACE` a good line separator? What happens if you have 2 words in your file with space between them? Maybe better to separate different lines using `LINE_SEPARATOR`?